From 08f4a9378a75d2d3b25ca1db3e1463f4fe929354 Mon Sep 17 00:00:00 2001
From: Kenny Kessler <kenny.kessler@motorola.com>
Date: Wed, 11 Nov 2015 12:12:42 -0600
Subject: [PATCH 469/959] IKSWM-8388: arm/dt: msm8952-moto-common: AFVC to 150
 mV

Set DTS with 150 mV of Float Voltage compensation.

Change-Id: Ib40e682f6aacebc7875baf5a8c4358362bf2b45b
Signed-off-by: Kenny Kessler <kenny.kessler@motorola.com>
Reviewed-on: http://gerrit.mot.com/798989
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Jason Knopsnyder <jasonk@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index 1545d95678d..5fa861be047 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -330,6 +330,7 @@
 	qcom,bmd-pin-src = "bpd_none";
 	qcom,usbid-disabled;
 	qcom,force-aicl-rerun;
+	qcom,auto-voltage-comp-mv = <150>;
 };
 
 /{
-- 
2.11.0

